core/domain/remote_access_services_test.py

```diff
@@ -25,15 +28,12 @@ class RemoteAccessServicesTests(test_utils.GenericTestBase):
 
     def test_that_generate_signature_works_correctly(self):
         """Test that generate signature function is working as expected."""
-        data = {
-            'vm_id': 'vm_default'
-        }
-
         with self.swap(vmconf, 'DEFAULT_VM_SHARED_SECRET', '1a2b3c4e'):
-            sign = remote_access_services.generate_signature(data)
+            sign = remote_access_services.generate_signature(
```

Comment:
Call it "signature", here and elsewhere. "sign" means something different.

In genera, try not to abbreviate, especially where you don't need to.

Reply:
Done